src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.ts

```diff
@@ -301,7 +303,15 @@ export class TableComponent implements AfterContentChecked, OnInit, OnChanges, O
 
   reloadData() {
     if (!this.updating) {
-      this.fetchData.emit();
+      this.loadingError = false;
+      const context = new CdTableFetchDataContext(() => {
+        this.loadingError = true;
+        // Force table to show no data. This will stop the loading
```

Comment:
If the connection is flaky and we fetch every 5 seconds per default, I think emptying the table is not the user friendliest idea.

Reply:
But displaying outdated and falsy data isn't an option, too.

Reply:
we should display the a warning that the data is older